lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_marriage_and_civil_partnership.govspeak.erb

```diff
@@ -34,7 +34,9 @@
                locals: { calculator: calculator } %>
 
   <% elsif calculator.ceremony_country == 'vietnam' %>
-    You’ll need to download and complete the following documents:
+    <% unless calculator.partner_is_national_of_ceremony_country?  %>
+      You’ll need to download and complete the following documents:
+    <% end %>
 
```

Comment:
@ikennaokpala As this text only ever refers to Vietnam's appointment documents, I would move it to partial `_appointment_documents_in_vietnam.govspeak.erb`